addons/main/modKonfigurationsTemplates/medicSettings/kat-disable.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * Should disable all KAT functionallity
+*/
+
+// KAT - ADV Medical: Airway
+force force kat_airway_enable = false;
+
+// KAT - ADV Medical: Breathing
+force kat_breathing_advPtxEnable = false;
```

Comment:
Intentionally not forcing for the mission?

Reply:
Should be fixed now